tests/data/app/db

```diff
@@ -1 +1 @@
-a:0:{}
+a:4:{s:6:"params";a:0:{}s:5:"query";a:0:{}s:4:"form";a:4:{s:8:"wireless";s:5:"mouse";s:6:"coffee";s:1:"0";s:3:"tea";s:1:"1";s:7:"vanilla";s:2:"on";}s:5:"files";a:0:{}}
```

Comment:
Could it be that these changes have been committed by mistake?

Reply:
It came up as a changed file after the tests ran so I presumed it was an older version of the file. I can revert if this file is meant to be an empty serialised array.

Reply:
It's supposed to be, revert the changes please

Reply:
Done, commit dropped. Thanks!